developer/services/admin-api.rst

```diff
@@ -355,9 +349,9 @@ it easy to set the policy using current mechanism.
    * - ``admin.vm.device.<class>.Assign``
      - vm
      - device
-     - assignement-serialization
+     - assignment-serialization
      - `-`
-     - ``device`` is in form ``<backend-name>+<device-ident>`` ``assignment-serialization`` is specified in the section Device Serialization.
+     - ``device`` is in form ``<backend-name>+<device-ident>``; ``assignment-serialization`` is specified in the section Device Serialization.
```

Comment:
Actually in the markdown version there is no ';' but a new line. I think we shoud leave with the markdown version. Would you be so nice to add newlines to such cells where applicable? (I can also do it afterwards. 
I think that ';' as a character may be misleading, though not likely. Thank you for the review of this importart doc!!